sdk/purview/purview-scanning-rest/src/models.ts

```diff
@@ -1935,7 +1793,7 @@ export type Scan =
   | AzureStorageMsiScan
   | TeradataCredentialScan
   | TeradataUserPassScan
-  | TeradataUserPassScanAutoGenerated
```

Comment:
AutoGenerated?

Reply:
still have autogen issues as duplicate schemas in their swagger